docker/dc.otel.dev.yml

```diff
@@ -0,0 +1,13 @@
+version: "3"
+services:
+
```

Comment:
Analogous to the backend, I would think that `OPENTELEMETRY_ENABLED` needs to be set to true for all services here